documentation/book/con-kafka-listeners.adoc

```diff
@@ -123,4 +123,48 @@ listeners:
 ----
 
 
-Authentication must be configured when using the User Operator to manage `KafkaUsers`.
\ No newline at end of file
+Authentication must be configured when using the User Operator to manage `KafkaUsers`.
+
+== Network Policies
+
+For every enabled listener, {ProductName} will automatically create a NetworkPolicy which will by default grant all applications access to the listener.
+You can restrict the access only to selected applications using the `networkPolicyPeers` field.
+You can use different `networkPolicyPeers` configuration for each listener.
+
+.An example where the `plain` and `tls` listeners allow connections only from selected pods
+[source,yaml,subs="attributes+"]
+----
+# ...
+listeners:
+      plain:
+        authentication:
+          type: scram-sha-512
+        networkPolicyPeers:
+          - podSelector:
+              matchLabels:
+                app: kafka-sasl-consumer
+          - podSelector:
+              matchLabels:
+                app: kafka-sasl-producer
+      tls:
+        authentication:
+          type: tls
+        networkPolicyPeers:
+          - namespaceSelector:
+              matchLabels:
+                project: myproject
+          - namespaceSelector:
+              matchLabels:
+                project: myproject2
+# ...
+----
+
+In the example above:
+
+* Only the application pods matching the labels `app: kafka-sasl-consumer` and `app: kafka-sasl-producer` which are running in the same namespace as the Kafka brokers can connect to the `plain` listener.
+* Only the application pods running in the namespaces matching the labels `project: myproject` and `project: myproject2` can connect to the `tls` listener.
+
+The syntax of the `networkPolicyPeers` field uses the same syntax as the `from` field in Kubernetes NetworkPolicies.
+For more information about the schema, see {K8sNetworkPolicyPeerAPI} and xref:type-KafkaListeners-reference[`KafkaListeners` schema reference].
+
+NOTE: To use NetworkPolicies, your {ProductPlatformName} needs to support Ingress NetworkPolicies.
```

Comment:
your {ProductPlatformName} > your version of {ProductPlatformName}
needs to > must

_Does this apply to a specific version? If so, can we state the versions of K8s and OpenShift?_

Reply:
This is not really about the version. This depends purely on the configuration / type of components used. Two cluster of the same version can support or not support the network policies.